src/js/utils/DOM.js

```diff
@@ -85,10 +85,15 @@ export const getBodyChildElements = () => {
   return children;
 };
 
-export const getNewContainer = (rootNode = document.body) => {
+export const getNewContainer = (rootNode = document.body, bodyTarget) => {
```

Comment:
I think `rootTarget` would be more aligned with `rootNode`. Although, I think `rootChildPosition` would be even more accurate.

Reply:
I like `rootChildPosition`. I'll use that.

Reply:
I actually liked your proposal above to use `targetChildPosition` and used that.